apps/authentication-portal/src/main/webapp/device-success.jsp

```diff
@@ -0,0 +1,104 @@
+<%--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+  ~
+  ~ WSO2 Inc. licenses this file to you under the Apache License,
+  ~ Version 2.0 (the "License"); you may not use this file except
+  ~ in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing,
+  ~ software distributed under the License is distributed on an
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  --%>
+
+<%@ page import="java.io.File" %>
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8"%>
+
+<%@include file="includes/localize.jsp" %>
+
+<!doctype html>
+<html>
+<head>
+    <!-- header -->
+    <%
+        File headerFile = new File(getServletContext().getRealPath("extensions/product-title.jsp"));
+        if (headerFile.exists()) {
+    %>
+    <jsp:include page="extensions/header.jsp"/>
+    <% } else { %>
+    <jsp:directive.include file="includes/header.jsp"/>
+    <% } %>
+</head>
+<body onload="loadFunc()">
+    <main class="center-segment">
+        <div class="ui container medium center aligned middle">
+            
+            <!-- product-title -->
+            <%
+                File productTitleFile = new File(getServletContext().getRealPath("extensions/product-title.jsp"));
+                if (productTitleFile.exists()) {
+            %>
+            <jsp:include page="extensions/product-title.jsp"/>
+            <% } else { %>
+            <jsp:directive.include file="includes/product-title.jsp"/>
+            <% } %>
+            <%if(request.getParameter("app_name") != null) { %>
+                <div class="field">
+                    <div class="ui positive message">
+                        <div class="ui header center aligned">
+                            Successful
+                        </div>
+                        <p>Login successful for application:
+                            <%= request.getParameter("app_name")%>.Please close the browser and
+                            return to your device.</p>
+                    </div>
+                </div>
+            <% } else { %>
+                <div class="ui negative message">
+                    <div class="header">
+                        Error
+                    </div>
+                <p>Requested parameter application name is not present.</p>
+                </div>
+            <% } %>
```

Comment:
Revert the `ui segment` wrapper which was there.

Reply:
done